src/modules/languages/r/default.nix

```diff
@@ -21,9 +25,23 @@ in
         description = "The radian package to use.";
       };
     };
+    descriptionFile = {
```

Comment:
What do you think if we go with `description.enable` that would also install the packages? Not sure it makes a lot of sense to enable it without doing anything.